### PR TITLE
Feature/cluster playbooks

### PR DESCRIPTION
**Pull request by takama**

### CODE REVIEW

File: playbooks/addons/setup.yml [hunk 1]
```diff
@@ -5,4 +5,5 @@
   become_method: sudo
```
Comment (STASiAN) [inline playbooks/addons/setup.yml]: privilege escalation method to use (default=sudo),

File: roles/dashboard/templates/deploy-dashboard.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+function deploy_dashboard {
+    if kubectl get deploy -l k8s-app=kubernetes-dashboard --namespace=kube-system | grep kubernetes-dashboard &> /dev/null; then
```
Comment (STASiAN) [inline roles/dashboard/templates/deploy-dashboard.sh]: > /dev/null 2>&1

File: roles/stats/templates/deploy-stats.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+function deploy_heapster {
+    if kubectl get deploy -l k8s-app=heapster --namespace=kube-system | grep heapster &> /dev/null; then
```
Comment (STASiAN) [inline roles/stats/templates/deploy-stats.sh]: > /dev/null 2>&1

File: roles/kubedns/templates/deploy-kubedns.sh [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    else
+        echo "Creating KubeDNS deployment"
+        kubectl apply -f {{ k8s_addons_dir }}/kubedns.yaml
+    fi
+
+  echo
+}
+
+function deploy_dns_autoscaler {
+    if kubectl get deploy -l k8s-app=kube-dns-autoscaler --namespace=kube-system | grep kube-dns-autoscaler &> /dev/null; then
```
Comment (STASiAN) [inline roles/kubedns/templates/deploy-kubedns.sh]: > /dev/null 2>&1

File: roles/kubedns/templates/deploy-kubedns.sh [hunk 5]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+function deploy_dns {
+    if kubectl get deploy -l k8s-app=kube-dns --namespace=kube-system | grep kube-dns &> /dev/null; then
```
Comment (STASiAN) [inline roles/kubedns/templates/deploy-kubedns.sh]: > /dev/null 2>&1

File: roles/addons/templates/deploy-addons.sh [hunk 6]
```diff
@@ -1,72 +1,5 @@
...
-    else
-        echo "Creating L7 Ingress controller"
-        kubectl apply -f {{ k8s_addons_dir }}/l7-ingress-controller.yaml
-    fi
-
-  echo
-}
-
 function deploy_cockroachdb {
     if kubectl get statefulset -l app=cockroachdb | grep cockroachdb &> /dev/null; then
```
Comment (STASiAN) [inline roles/addons/templates/deploy-addons.sh]: > /dev/null 2>&1

File: roles/dashboard/templates/deploy-dashboard.sh (hunk 2 above)
Comment (takama) [reply roles/dashboard/templates/deploy-dashboard.sh]: fixed

File: roles/kubedns/templates/deploy-kubedns.sh (hunk 5 above)
Comment (takama) [reply roles/kubedns/templates/deploy-kubedns.sh]: fixed

File: roles/stats/templates/deploy-stats.sh (hunk 3 above)
Comment (takama) [reply roles/stats/templates/deploy-stats.sh]: fixed

File: roles/kubedns/templates/deploy-kubedns.sh (hunk 4 above)
Comment (takama) [reply roles/kubedns/templates/deploy-kubedns.sh]: fixed

File: roles/addons/templates/deploy-addons.sh (hunk 6 above)
Comment (takama) [reply roles/addons/templates/deploy-addons.sh]: fixed

**Review by STASiAN — APPROVED** [review]

**Pull request merged**